src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import commands.ByeCommand;
+import commands.Command;
+import commands.DeadlineCommand;
+import commands.DeleteCommand;
+import commands.EventCommand;
+import commands.ListCommand;
+import commands.MarkCommand;
+import commands.TodoCommand;
+import commands.UnMarkCommand;
+import dukeexceptions.DukeException;
+import dukeexceptions.NoDescriptionException;
+import dukeexceptions.NoSuchCommandException;
+import tasks.TaskList;
+
+
+public class Duke {
```

Comment:
Could add in JavaDoc for this class

Reply:
Agreed. Perhaps more JavaDoc and header comments will increase readability?